components/NavigationBar/index.js

```diff
@@ -5,33 +5,42 @@ import { FaHouse, FaPlus, FaChartPie } from "react-icons/fa6";
 function NavigationBar() {
   const router = useRouter();
 
-  const isActive = (path) => {
-    return router.pathname === path;
-  };
+ function isActive(path) {
+   if (!path || path == undefined) {
```

Comment:
the second condition will never be tested. 
```suggestion
   if (!path) {
```